Source/DafnyCore/AST/Tokens.cs

```diff
@@ -204,6 +204,15 @@ public int CompareTo(IToken other) {
   public int CompareTo(Boogie.IToken other) {
     return WrappedToken.CompareTo(other);
   }
+
+  public static IToken Unwrap(IToken token, bool includeRanges = false) {
```

Comment:
```suggestion
  /// Removes token wrappings from a given token, so that it returns the bare token
  public static IToken Unwrap(IToken token, bool includeRanges = false) {
```
Or something similar. I like docstrings on functions just to give the intent